rskj-core/src/test/java/co/rsk/rpc/netty/Web3WebSocketServerTest.java

```diff
@@ -203,8 +203,13 @@ private byte[] getJsonRpcDummyMessage(String value) {
 
         byte[] request = new byte[0];
         try {
-            request = OBJECT_MAPPER.writeValueAsBytes(OBJECT_MAPPER.treeToValue(
-                    JSON_NODE_FACTORY.objectNode().setAll(jsonRpcRequestProperties), Object.class));
+            Object object = OBJECT_MAPPER.treeToValue(JSON_NODE_FACTORY.objectNode().setAll(jsonRpcRequestProperties), Object.class);
+
+            if (object == null) {
+                throw new NullPointerException();
```

Comment:
I wouldn't throw exceptions in a Test, I would do nothing here actually, its under our control and this should not be reached but for our mistake.

Reply:
Not necessary anymore with the latest changes